solc/CommandLineInterface.cpp

```diff
@@ -1638,18 +1638,6 @@ void CommandLineInterface::handleAst(string const& _argStr)
 		vector<ASTNode const*> asts;
 		for (auto const& sourceCode: m_sourceCodes)
 			asts.push_back(&m_compiler->ast(sourceCode.first));
-		map<ASTNode const*, evmasm::GasMeter::GasConsumption> gasCosts;
```

Comment:
This wasn't even used/displayed by anything? Weird.

Reply:
Another call for https://github.com/ethereum/solidity/issues/9583 ;-).